pkg/pillar/cmd/zedmanager/zedmanager.go

```diff
@@ -467,8 +448,10 @@ func lookupAppInstanceConfig(ctx *zedmanagerContext, key string) *types.AppInsta
 	return &config
 }
 
-func handleCreate(ctx *zedmanagerContext, key string,
-	config types.AppInstanceConfig) {
+func handleAppInstanceConfigCreate(ctxArg interface{}, key string,
```

Comment:
Why change its name? Then the log calls don't match the function name any more.

Reply:
I suggested to change the name so that there is no confusion on which Object type this routine belongs.. There are multiple subscriptions in the file - so, clearer if we have an appropriate prefix

May be update the log statements to reflect the new name?

Reply:
@kalyan-nidumolu I disagree with piecemeal renaming; a single function or a single object on a single agent. That just results in inconsistencies which makes it harder to read the code.

Renaming all handle*Create/Modify/Delete to handle<object>Create/Modify/Delete in all the agents would make sense.